fendermint/testing/contracts/QueryBlockhash.sol

```diff
@@ -0,0 +1,10 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+contract QueryBlockhash {
```

Comment:
@fridrik01 If I'm not mistaken, this is your first Solidity contract, right? 🆒

Reply:
yebb